packages/editor-ui/src/components/InlineExpressionEditor/InlineExpressionEditorInput.vue

```diff
@@ -127,6 +144,9 @@ export default mixins(expressionManager, workflowHelpers).extend({
 		focus() {
 			this.editor?.focus();
 		},
+		onClose() {
+			this.ndvStore.activeNodeName = null;
```

Comment:
For the expression editor modal, it was requested that Esc with completions menu open dismisses the menu, and another Esc closes the modal. So by extension, I thought the same should apply to NDV, which is also a modal. Let me check with Max.

Reply:
found it confusing.. anyways, this is problematic here. what if you open NDV then a credential, pressing esc there will close node but not credential modal.. should emit an event here and let parent decide how to behave

Reply:
Got it, for simplicity and since adding this was not explicitly requested, I'll just remove it.

Reply:
I'm not sure I have full context here since we're looking at a function and not a flow, so pardon verbosity:
- Gold standard UX I see for "Esc" in this context comes from Figma. Each "esc" performs a blur or close action on current thing focused and then moves up a level. E.g. if you're focused on a param in ndv; first esc should blur, second esc should close the modal.
- So in inline expressions: 1st esc escapes the autocomplete list, second esc escapes the param (blur), third esc closes ndv.

This is all just my gold standard rec; without knowing complexity of doing so. Hope that helps.

Reply:
Thanks @maxtkacz. Since this detailed dismiss-blur-close flow for `Esc` behavior did not come up before either here or [there](https://linear.app/n8n/issue/ADO-188/dismissing-completion-and-closing-expression-editor-modal-share-esc), let's discuss later if it fits in the next phase. Let me know if you disagree.